Makefile

```diff
@@ -1,15 +1,12 @@
 PKG := github.com/btcsuite/btcd
 
 LINT_PKG := github.com/golangci/golangci-lint/cmd/golangci-lint
-GOACC_PKG := github.com/ory/go-acc
 GOIMPORTS_PKG := golang.org/x/tools/cmd/goimports
 
 GO_BIN := ${GOPATH}/bin
 LINT_BIN := $(GO_BIN)/golangci-lint
-GOACC_BIN := $(GO_BIN)/go-acc
 
 LINT_COMMIT := v1.18.0
-GOACC_COMMIT := 80342ae2e0fcf265e99e76bcc4efd022c7c3811b
```

Comment:
So this is important, as it absolves any linter violations _before_ that commit. Otherwise, we'll have to have a PR which fixes _every_ linter violation. 

In `lnd`, we use `golangci` which still supports this feature: https://github.com/lightningnetwork/lnd/pull/9382/files